TaskManagement/accounts/views.py

```diff
@@ -27,8 +30,13 @@ class UserDetails(ListView):
     """show the details of user"""
     model = User
     template_name = 'account/userdetails.html'
-    paginate_by = 6
-    ordering = ['username']
+    paginate_by = 7
+    ordering = ['-id']
+    # def get_queryset(self):
+    #     search =self.request.GET.get('search_here')
```

Comment:
remove commented code if not required

Reply:
all comments are already removed in accounts app...this code was outdated